ios/FluentUI.Demo/FluentUI.Demo/Demos/ActivityIndicatorDemoController.swift

```diff
@@ -236,45 +236,39 @@ extension ActivityIndicatorDemoController: DemoAppearanceDelegate {
         guard let fluentTheme = self.view.window?.fluentTheme else {
             return
         }
-        if isOverrideEnabled {
-            fluentTheme.register(controlType: ActivityIndicator.self, tokens: {
-                ThemeWideOverrideActivityIndicatorTokens()
-            })
-        } else {
-            fluentTheme.register(controlType: ActivityIndicator.self, tokens: nil)
-        }
-
+        fluentTheme.register(tokenSetType: ActivityIndicatorTokenSet.self,
+                             tokenSet: isOverrideEnabled ? themeWideOverrideActivityIndicatorTokens : nil)
     }
 
     func perControlOverrideDidChange(isOverrideEnabled: Bool) {
         defaultColorIndicators.values.forEach { activityIndicator in
-            activityIndicator.state.overrideTokens = (isOverrideEnabled ? PerControlOverrideActivityIndicatorTokens() : nil)
+            perControlOverrideActivityIndicatorTokens.forEach { (key, value) in
+                if isOverrideEnabled {
+                    activityIndicator.tokenSet[key] = value
+                } else {
+                    activityIndicator.tokenSet.removeOverride(key)
```

Comment:
Bad idea and probably not worth it, but thought I'd share anyway.
Should this have stack-like behavior? So that if you add an override, then remove it, you don't completely reset any pre-existing overrides?

Reply:
Each token value (`.backgroundColor`, `.borderWidth`, etc) can only ever have one direct override value. This removal API simply removes the specific override for that token, without affecting any other token value.

I don't think we want to support any sort of FILO-style "queueing" of tokens, if that's what you're suggesting. Our intent is to support exactly two approaches to overrides:

- Direct overrides via modifying the `tokenSet` for a given component.
- Indirect overrides via the `FluentTheme` registration API.

Or am I misunderstanding your question?

Reply:
No, you got the question. I figured we didn't want to have any sort of token history, but figured I'd ask since I had the thought.